### ISSUE TITLE
Tenya won't accept bowl

### ISSUE THREAD

**Issue by enderandrew** [EE-Mod-Setup/EE-Mod-Setup#90]
I meet three fishermen who want me to kill a priestess. Sandrah immediately steps in to object saying she knows they stole a bowl. The text said they were going to give me the bowl bit didn't. I talked to them a second time and they did give me the bowl. I went up to the next map and met Tenya who does not give me any option to return the bowl. In vanilla she is supposed to attack me, but she never attacks me. If I attack her, she doesn't fight back and dies, even if I attack her slowly with low damage. I have the bowl and no way to progress the quest.
[EE-Debug.zip](https://github.com/EE-Mod-Setup/EE-Mod-Setup/files/12270536/EE-Debug.zip)

**Comment by RoxanneSHS** [comment]
See what C:GetGlobal("Umbowl","Global") returns. It should be 2 if Sandrah intervened with the fisherman and started a dialogue afterwards to apologize for her behaviour. This should give you a response for Tenya to hand her the bowl (provided you still have it in the inventory. "Misc53")

**Comment by enderandrew** [comment]
It is 2 currently. If I set it to 1, Sandrah jumps in to apologize again, and it goes back to 2. I tried talked to Tenya again, and still no luck. I also experimented setting it to 3 or 4 and still no luck. There are multiple mods that touched Tenya in my install. Is there a way to see what other mods may have caused an issue? Or is there another global value that should affect her accepting the bowl?

https://imgur.com/a/PLr0FBN
